samples/user-interface/predictiveback/README.md

```diff
@@ -207,3 +208,33 @@ class MyFragment : Fragment() {
 }
 ```
 
+## Cross-fragment animation with MaterialSharedAxis
+
+MaterialSharedAxis is a Visibility transition. A Visibility transition is triggered when the target
+View's visibility is changed or when the View is added or removed. This means MaterialSharedAxis
```

Comment:
Let's say fragments instead of views, since this is a cross-fragment transition.

Reply:
Done! TY!